### PR TITLE
Changes for resizer image and initcontainer

### PR DESCRIPTION
**Pull request by ambiknai**

### CODE REVIEW

File: assets/rbac/initcontainer_rolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+ kind: ClusterRoleBinding
+ apiVersion: rbac.authorization.k8s.io/v1
+ metadata:
+   name: vpc-block-initcontainer-binding
+   labels:
+     app: ibm-vpc-block-csi-driver
+     addonmanager.kubernetes.io/mode: Reconcile
+   annotations:
+      version: 2.0.3_354
```
Comment (arahamad-zz) [inline assets/rbac/initcontainer_rolebinding.yaml]: please remove this for other objects as well

File: assets/rbac/initcontainer_role.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   name: vpc-block-initcontainer-role
+   labels:
+     app: ibm-vpc-block-csi-driver
+     addonmanager.kubernetes.io/mode: Reconcile
+   annotations:
+     version: 2.0.3_354
+ rules:
+   - apiGroups: [""]
+     resources: ["nodes"]
+     verbs: ["get", "list", "watch", "update"]
```
Comment (arahamad-zz) [inline assets/rbac/initcontainer_role.yaml]: watch is not required

**Review by meghnasiingh — APPROVED** [review]

**Review by arahamad-zz — CHANGES_REQUESTED** [review]
pl check comments

File: assets/rbac/initcontainer_role.yaml (hunk 2 above)
Comment (ambiknai) [reply assets/rbac/initcontainer_role.yaml]: Done

**Comment by ambiknai** [comment]
```
ambikanair@Ambika-Nairs-MacBook-Pro vpc-node-label-updater % oc get pods  -n openshift-cluster-csi-drivers
NAME                                            READY   STATUS    RESTARTS   AGE
ibm-vpc-block-csi-controller-54cb8444b8-qfvt8   5/5     Running   0          105s
ibm-vpc-block-csi-node-7m4fv                    3/3     Running   0          105s
ibm-vpc-block-csi-node-8c7fn                    3/3     Running   0          105s
ibm-vpc-block-csi-node-j2pv6                    3/3     Running   0          105s
ibm-vpc-block-csi-node-mdf6h                    3/3     Running   0          105s
ibm-vpc-block-csi-node-ng7dk                    3/3     Running   0          105s
ibm-vpc-block-csi-node-qtvl7                    3/3     Running   0          105s
```

```
  ----    ------          ----  ----               -------
  Normal  Scheduled       11m   default-scheduler  Successfully assigned openshift-cluster-csi-drivers/ibm-vpc-block-csi-controller-54cb8444b8-qfvt8 to ipi-dev-storage-7-vfvrb-worker-1-5z85k
  Normal  AddedInterface  11m   multus             Add eth0 [10.131.0.61/23] from openshift-sdn
  Normal  Pulling         11m   kubelet            Pulling image "k8s.gcr.io/sig-storage/csi-resizer:v1.2.0"
  Normal  Pulled          11m   kubelet            Successfully pulled image "k8s.gcr.io/sig-storage/csi-resizer:v1.2.0" in 562.787616ms
  Normal  Created         11m   kubelet            Created container csi-resizer
  Normal  Started         11m   kubelet            Started container csi-resizer
  Normal  Pulling         11m   kubelet            Pulling image "k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2"
  Normal  Pulled          11m   kubelet            Successfully pulled image "k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2" in 480.092288ms
  Normal  Created         11m   kubelet            Created container liveness-probe
  Normal  Started         11m   kubelet            Started container csi-provisioner
  Normal  Created         11m   kubelet            Created container csi-provisioner
  Normal  Pulled          11m   kubelet            Successfully pulled image "k8s.gcr.io/sig-storage/csi-attacher:v3.2.1" in 477.49053ms
  Normal  Created         11m   kubelet            Created container csi-attacher
  Normal  Started         11m   kubelet            Started container csi-attacher
  Normal  Pulled          11m   kubelet            Container image "k8s.gcr.io/sig-storage/livenessprobe:v2.3.0" already present on machine
  Normal  Pulling         11m   kubelet            Pulling image "k8s.gcr.io/sig-storage/csi-attacher:v3.2.1"
  Normal  Started         11m   kubelet            Started container liveness-probe
  Normal  Pulling         11m   kubelet            Pulling image "gcr.io/k8s-staging-cloud-provider-ibm/ibm-vpc-block-csi-driver:master"
  Normal  Pulled          11m   kubelet            Successfully pulled image "gcr.io/k8s-staging-cloud-provider-ibm/ibm-vpc-block-csi-driver:master" in 477.362192ms
  Normal  Created         11m   kubelet            Created container iks-vpc-block-driver
  Normal  Started         11m   kubelet            Started container iks-vpc-block-driver
ambikanair@Ambika-Nairs-MacBook-Pro vpc-node-label-updater % 

```

**Review by arahamad-zz — APPROVED** [review]
lgtm

**Review by akgunjal — APPROVED** [review]
LGTM

**Pull request merged**